core/src/test/scala/org/apache/spark/rdd/LocalCheckpointSuite.scala

```diff
@@ -168,6 +172,10 @@ class LocalCheckpointSuite extends SparkFunSuite with LocalSparkContext {
     // Collecting the RDD should now fail with an informative exception
     val blockId = RDDBlockId(rdd.id, numPartitions - 1)
     bmm.removeBlock(blockId)
+    // Wait until the block has been removed successfully.
+    eventually(timeout(1 seconds), interval(100 milliseconds)) {
+      assert(bmm.getBlockStatus(blockId).size == 0)
```

Comment:
@srowen, actually, do you maybe know why `===` is preferred over `==`? Up to my knowledge, `===` is preferred as it gives a better error message and I saw several comments saying so by few committers before.

However, it seems some do not think so. I raised an issue before - https://github.com/databricks/scala-style-guide/issues/36 but I am still confused.